examples/api/lib/material/app_bar/sliver_app_bar.4.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [SliverAppBar].
+
+void main() {
+  runApp(const StretchableSliverAppBar());
+}
+
+class StretchableSliverAppBar extends StatefulWidget {
+  const StretchableSliverAppBar({super.key});
+
+  @override
+  State<StretchableSliverAppBar> createState() =>
+      _StretchableSliverAppBarState();
+}
+
+class _StretchableSliverAppBarState extends State<StretchableSliverAppBar> {
+  bool _stretch = true;
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        home: Scaffold(
+      body: CustomScrollView(
+        physics: const BouncingScrollPhysics(),
+        slivers: <Widget>[
+          SliverAppBar(
+            stretch: _stretch,
+            onStretchTrigger: () async {
+              // Triggers when stretching
```

Comment:
When stretching to the extent of `stretchTriggerOffset`, this would be nice to clarify or add to the sample.